config/indexer_settings_test.yml

```diff
@@ -2,4 +2,6 @@
 
 processing_thread_pool: 5
 hathi_overlap_path: spec/fixtures/hathitrust/
-hathi_overlap_file: mock_overlap.csv
\ No newline at end of file
+hathi_mono_overlap_file: mock_mono_overlap.csv
```

Comment:
Same thing for the files, I only ass values to env indexer setting files if they are different from the default.

Reply:
🤔 I added these because they are different from the default ones, so sorry I think I do not understand what change you request/recommend here?

Reply:
maybe you meant to write this to the dev file? I can remove the ones in the dev file.

Reply:
oh I'm so sorry! I must have read that wrong. :(